src/crg_imprim.jl

```diff
@@ -257,3 +257,43 @@ where ``N`` is the number of reflections, ``N^*`` is the number of reflecting hy
 function coxeter_number(W::ImprimitiveComplexReflectionGroup)
 	return div(num_reflections(W)+num_hyperplanes(W),ZZ(rank(W)))
 end
+
+
+"""
+	num_conjugacy_classes(W::ImprimitiveComplexReflectionGroup)
+
+The number of conjugacy classes of an Imprimitive Complex Reflection Group ``W`` is equal to the number of multipartitions if ``p=1``, otherwise do a weighted sum of the number of ``k``-stuttering ``m``-multipartitions of ``n`` with ``k \\mid gcd(n,p)``. A ``k``-stuttering multipartition ``\\lambda=(\\lambda^{(0)}, ..., \\lambda^{(m-1)})`` has the property ``\\lambda^{(i)} = \\lambda^{(i+k)}`` where the indexing is modulo ``m``.
+Since this is also the number of irreducible complex representations of ``W``, one can refer to J. Stembridge [On the Eigenvalues of Representations of Reflection Groups and Wreath Products](https://msp.org/pjm/1989/140-2/pjm-v140-n2-p06-p.pdf), section 6.
+"""
```

Comment:
I would suggest to also give a source/reference  for the algorithm you implement